ortools/constraint_solver/java/routing.i

```diff
@@ -102,18 +98,11 @@ import java.util.function.LongUnaryOperator;
 %ignore RoutingModel::PackCumulsOfOptimizerDimensionsFromAssignment;
 %ignore RoutingModel::RegisterStateDependentTransitCallback;
 %ignore RoutingModel::RemainingTime;
-%ignore RoutingModel::SolveWithParameters(
-    const RoutingSearchParameters& search_parameters,
-    std::vector<const Assignment*>* solutions);
```

Comment:
Is this supported ? (the vector* as parameter)

Reply:
IIRC no, that's why it is currently not wrapped because it will lead to tons of swig PINVOKE opaque pointer making this API ugly and unusable.

Reply:
I will revert to ignore that one and SolveFromAssignmentWithParameters with the vector<const Assignment*>* solutions argument as well